lib/responses.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = Object.freeze({
+    AUTHENTICATED: {
+        code: 'AUTHENTICATED',
+        message: 'Successfully authenticated'
+    },
+
+    UNAUTH: {
+        code: 'UNAUTH',
+        message: 'Unauthorized access'
+    },
+
+    LOGGED_OUT: {
+        code: 'LOGGED_OUT',
+        message: 'Successfully logged out'
+    },
+
+    OAUTH_ERROR: {
+        code: 'OAUTH_ERROR',
+        message: 'Error occurred in getting OAUTH access token from Freedom! server'
```

Comment:
possibly not from freedom server now that we have customers. Let us make this message a bit generic now.

Reply:
Changed this to general message.